avionics/statemachine.cpp

```diff
@@ -60,7 +60,7 @@ void stateMachine(float *altitude, float *delta_altitude, float *prev_delta_alti
             }
             else{
                 launch_count = 0;
```

Comment:
Nothing re: code here but we don't have code that changes our sampling interval.  I'd imagine it would go here and our timing choice will effect our ground altitude array size.